### PR TITLE
Release 1.10.0

### PR DESCRIPTION
**Pull request by mrclay**
- [x] mark #7677 fixed after merging

cc @Elgg/owners

### CODE REVIEW

**Comment by juho-jaakkola** [comment]
A couple of tests seem to be failing.

**Comment by Srokap** [comment]
Where? https://travis-ci.org/Elgg/Elgg/builds/46678734

**Comment by juho-jaakkola** [comment]
The list of commits shows errors at 8e577be but apparently Github just isn't in sync with the latest results.

**Comment by Srokap** [comment]
For me it's only relevant to look at last commit CI status as if something passes after merge, it's fine regardless of what happened for particular previous commits.

**Comment by mrclay** [comment]
The two merge commits LGTM

**Comment by jdalsem** [comment]
don't forget the todo on this ticket...

File: engine/lib/comments.php [hunk 1]
```diff
@@ -193,3 +194,36 @@ function _elgg_comments_permissions_override($hook, $type, $return, $params) {
...
+ * @param array  $returnvalue Current mail parameters
+ * @param array  $params      Original mail parameters
+ * @return array $returnvalue Modified mail parameters
+ */
+function _elgg_comments_notification_email_subject($hook, $type, $returnvalue, $params) {
+
+	/** @var Elgg_Notifications_Notification */
+	$notification = elgg_extract('notification', $returnvalue['params']);
+
+	if ($notification instanceof Elgg_Notifications_Notification) {
```
Comment (mrclay) [inline engine/lib/comments.php]: Forgot to change these to namespaces, so this isn't working in 1.10.0. I'm looking for more serious issues...

Comment (Srokap) [inline engine/lib/comments.php]: We could use some test that would analyze code for this issue on CI.

Comment (mrclay) [inline engine/lib/comments.php]: fix #7783. We should definitely watch out for anything like "Elgg_[A-Z]" while merging into 1.10

**Pull request merged**